src/Strings.f90

```diff
@@ -287,42 +285,42 @@ MODULE Strings
   !> character strings.
   INTERFACE OPERATOR(//)
-    !> @copybrief Strings::concatenate_char_onto_StringType
-    !> @copydetails Strings::concatenate_char_onto_StringType
+    !> @copybrief StringType::concatenate_char_onto_StringType
+    !> @copydetails StringType::concatenate_char_onto_StringType
     MODULE PROCEDURE concatenate_char_onto_StringType
-    !> @copybrief Strings::concatenate_StringType_onto_char
-    !> @copydetails Strings::concatenate_StringType_onto_char
+    !> @copybrief StringType::concatenate_StringType_onto_char
+    !> @copydetails StringType::concatenate_StringType_onto_char
     MODULE PROCEDURE concatenate_StringType_onto_char
-    !> @copybrief Strings::concatenate_StringType_onto_StringType
-    !> @copydetails Strings::concatenate_StringType_onto_StringType
+    !> @copybrief StringType::concatenate_StringType_onto_StringType
+    !> @copydetails StringType::concatenate_StringType_onto_StringType
     MODULE PROCEDURE concatenate_StringType_onto_StringType
   ENDINTERFACE
 
   !> @brief Overloads the Fortran intrinsic operator for comparing
   !> two variables to see if they are equal
   INTERFACE OPERATOR(==)
-    !> @copybrief Strings::equalto_char_StringType
-    !> @copydetails Strings::equalto_char_StringType
+    !> @copybrief StringType::equalto_char_StringType
+    !> @copydetails StringType::equalto_char_StringType
     MODULE PROCEDURE equalto_char_StringType
-    !> @copybrief Strings::equalto_StringType_char
-    !> @copydetails Strings::equalto_StringType_char
+    !> @copybrief StringType::equalto_StringType_char
+    !> @copydetails StringType::equalto_StringType_char
     MODULE PROCEDURE equalto_StringType_char
-    !> @copybrief Strings::equalto_StringType_StringType
-    !> @copydetails Strings::equalto_StringType_StringType
+    !> @copybrief StringType::equalto_StringType_StringType
+    !> @copydetails StringType::equalto_StringType_StringType
     MODULE PROCEDURE equalto_StringType_StringType
   ENDINTERFACE
 
   !> @brief Overloads the Fortran intrinsic operator for comparing
   !> two variables to see if they are not equal
   INTERFACE OPERATOR(/=)
-    !> @copybrief Strings::notequalto_char_StringType
-    !> @copydetails Strings::notequalto_char_StringType
+    !> @copybrief StringType::notequalto_char_StringType
+    !> @copydetails StringType::notequalto_char_StringType
     MODULE PROCEDURE notequalto_char_StringType
-    !> @copybrief Strings::notequalto_StringType_char
-    !> @copydetails Strings::notequalto_StringType_char
+    !> @copybrief StringType::notequalto_StringType_char
+    !> @copydetails StringType::notequalto_StringType_char
     MODULE PROCEDURE notequalto_StringType_char
-    !> @copybrief Strings::notequalto_StringType_StringType
-    !> @copydetails Strings::notequalto_StringType_StringType
+    !> @copybrief StringType::notequalto_StringType_StringType
+    !> @copydetails StringType::notequalto_StringType_StringType
```

Comment:
Hmm... does doxygen expect the type name or the module name?  I guess I'm not 100% clear on that...

Reply:
I thought it was the type name, but I guess the style guide says module name.

Reply:
Ok, thanks for checking.  Let's get that switched back.  I remember asking about that when we finalized the style guide, but didn't remember which way it went.